.github/dependabot.yml

```diff
@@ -0,0 +1,133 @@
+version: 2
+updates:
+  - package-ecosystem: "github-actions"
+    directory: "/build-image"
+    schedule:
+      interval: "daily"
```

Comment:
make these weekly, daily seems a bit excessive. I would even go for monthly. At least weekly.